views/browse.erb

```diff
@@ -22,7 +22,7 @@
     <ul>
     <% @inorganic_categories.each do | cat | %>
       <li>
-          <a href=<%= "/#{@mechanism}/inorganic?category=#{URI::encode(cat)}" %>>
+          <a href=<%= "/#{@mechanism}/inorganic?category=#{CGI::escapeURIComponent(cat)}" %>>
```

Comment:
Did this change come from a Ruby 2 -> 3 migration recommendation?

Do these functions work identically?

Reply:
It was a recommendation I found [here](https://rubyreferences.github.io/rubychanges/3.2.html) (`URI.encode` = `URI.escape`). They don't behave 100% identically as `URI.encode` was more generic and designed to escape entire URLs, while `CGI.escapeURIComponent` is more specific and exactly designed for the use-case we have here, where we want to escape a portion of text to allow it to be entered into a URL